### DISCUSSION TITLE
Find a hidden number

### DISCUSSION THREAD

**Discussion by ParanoidUser** [ParanoidUser/yolo#890] [Galaxy Trivia] [answered]
<table>
      <tr>
         <th colspan="2">12</th>
         <th colspan="2">35</th>
         <th colspan="2">?</th>
      </tr>
      <tr>
         <th>7</th>
         <th>3</th>
         <th>9</th>
         <th>2</th>
         <th>6</th>
         <th>2</th>
      </tr>
      <tr>
         <th>4</th>
         <th>1</th>
         <th>8</th>
         <th>3</th>
         <th>8</th>
         <th>7</th>
      </tr>
   </table>

**Comment by peterbaccaro** [comment, marked as answer]
The hidden number is **4**.

The number is calculated by deducting the right number from the left number in each row an then multiplying the numbers together.

- **6 - 2 = 4**
- **8 - 7 = 1**
- **4 x 1 = 4**
